Basal area updated 2023

arunherb opened the pull request:
Updated the methods vocabs mapping as per the July 2023 EMSA protocols.

The `full DBH` and `lite DBH` are integrated as `DBH`. This PR removed previous sub protocols, and created new Concept / Method `DBH`.

junrongYu commented:
Hi @arunherb , basal wedge protocol missed figure 2. Also, figure 5 & 6 are broken, could you have a look? Please run `git pull` after switching to this branch.

arunherb commented:
Hi Jun, 

Thanks for reviewing. I'll have a look.

arunherb commented:
@junrongYu - I have fixed the gaps.
* Added the Figure 2
*  Updated the broken links for 5 and 6.
Please have another review and let me know if all looks okay.
